pkg/module_manager/loader/fs/fs.go

```diff
@@ -72,7 +74,7 @@ func (fl *FileSystemLoader) getBasicModule(definition moduleDefinition, commonSt
 	return m, nil
 }
 
-// reads single directory and returns BasicModule
+// LoadModule reads single directory and returns BasicModule
```

Comment:
Prefferable to delete useless comment, not to refactor it.
Whats purpose of this comment?

It looks obvious.
LoadModule by given path and return BasicModule.

Reply:
LoadModule is a public function, which means it must contain a docstring to generate documentation on pkg.go.dev

Another question is that the documentation should be more detailed. But at a minimum, its writing must meet the standards.

Reply:
1) Addon operator do not used as a library from external developers. 
2) We do not need to cover clear code by obvious comments.

Reply:
This is where our opinions differ :)

Reply:
I think not only mine is different. We have a lot of [codebase](https://github.com/deckhouse/dmt/blob/main/internal/flags/flags.go) without comment coverage.
Looks like we need to start apply practices, that we'll bring in discussions.